src/harness/testdata/exz_record_1.json

```diff
@@ -0,0 +1,47 @@
+{
+  "id"  : "exclusion_zone/ntia/2018_01_10",
```

Comment:
This is not the format defined in the testing API doc.

Reply:
Updated   the format to zone/exclusion_zone/ntia/2018_01_09/exz_zone1

Reply:
This entire file does not match the format defined in the testing API doc. Please check again.

Reply:
Please also note that this file appears to be in the wrong location (but my comment applies to the other EXZ json files as well).

Reply:
The format of exz_record_0.json is correct but the format of this file is incorrect. Please update.